10.10 hwa / transcode docs update (#1120)
jellyfin/jellyfin.org: docs/general/administration/hardware-selection.md

Intel Drivers are much easier to install on Linux, with many distributions including them by default. If you do not need CUDA for other applications, it is highly recommended that you stick with Intel Graphics on Linux.

### Servers without GPUs

> Not having a GPU is **NOT** recommended for Jellyfin, as video transcoding on the CPU is very performance demanding. HDR to SDR tone-mapping can make the situation even worse. Depending on your configuration, you may end up in situations where a Ryzen 9 5950X cannot handle even a single video stream. Please read the detailed section below.

Reviewer: ```suggestion
Not having a GPU is **NOT** recommended for Jellyfin, as video transcoding on the CPU is very performance demanding. HDR to SDR tone-mapping can make the situation even worse. Depending on your configuration, you may end up in situations where a Ryzen 9 5950X cannot handle even a single video stream. Please read the section below for more details.
```

Maybe link to the section directly.

Intel Drivers are much easier to install on Linux, with many distributions including them by default. If you do not need CUDA for other applications, it is highly recommended that you stick with Intel Graphics on Linux.

### Servers without GPUs

Not having a GPU is **NOT** recommended for Jellyfin. The following specs are provided for reference only, for software tone-mapping 4K content to SDR:

> - CPU: AMD Ryzen 9 5950X, Intel Core i7-12700K or better

Reviewer 2: This only applies to Dolby Vision, videos does not require dolby vision reshaping do not need 16core zen3, and the 16core requirement is 60fps playback, so of the goal is 24fps the requirement can also be lower. Also, if users are having 12700K they can just use the integrated graphics.

The requires CPU power differs vastly for use cases and probably should not be documented like this because 4K60 DV tone mapping + transcoding is probably the most demanding task.

Intel Drivers are much easier to install on Linux, with many distributions including them by default. If you do not need CUDA for other applications, it is highly recommended that you stick with Intel Graphics on Linux.

### Servers without GPUs

Not having a GPU is **NOT** recommended for Jellyfin. The following specs are provided for reference only.

<details>
- RAM: 16GB

For software tone-mapping 4K 24fps Dolby Vision content to SDR:
CPU: AMD Ryzen 7 5700X, Intel Core i5-12400F or better

For 4K 60fps Dolby Vision content:
CPU: AMD Ryzen 9 5950X, Intel Core i7-12700F or better

> </details>

Reviewer 2: I'd say this is still inaccurate. 12700F has no way to guarantee 4K60 dolby vision and choosing CPU solely by dolby vision content performance is still very biased because as I said earlier it is probably the worst thing we can have. Dolby Vision videos has no fallback (like profile 5 videos) will have a significant impact on performance to a point that you will need 8 core zen3 for 4K24 playback while normal HDR10 videos or dolby vision videos with fallback (like profile 8.1) can be tone mapped with only 4 core Zen3 and still get 4K24 playback. These numbers are encoding with libx264 veryfast, using libx265 or libsvtav1 may reduce the framerate even further, so don't try to make a "recommended spec" because we cannot as it is way too scenario specific. 

We also need to be clear about the following points why CPU only is not recommended:

- Transcoding videos with CPU is performance demanding, and it is hard to gurantee all clients are compatible with all media sources
- Advanced codecs like HEVC and AV1 is even slower
- HDR to SDR tone mapping can make things worse, depending on the inputs you may end up with need 16 core zen3 level performance, or even no desktop CPUs currently can handle(if you want to use libx265 for dolby vision 4K60)

So if we have to do hardware selection guide on CPUs, we just say as much CPU cores as possible because the range is just too wide. The CPU only needs raspberry pi level performance if no video transcoding is needed, and a 4-6 core zen 3 can handle most use cases, including HDR10 4k24 tone mapping with libx264 encoder, but the CPU requirement goes up significantly when libx265/libsvtav1 is used and/or dolby vision reshaping is required.

- Intel "Atom" CPUs: Intel J/M/N/Y series low power CPUs up to 11th gen use a different architecture than higher end parts, leading to subpar performance despite what their names might suggest. Please be careful about these parts when shopping for a system to run Jellyfin.
- Prebuilt NAS devices: The software environment on most Prebuilt NAS devices often causes 3rd party software to be hard to install and not work properly. They also often have low end processors that are too slow for a good Jellyfin experience (Intel Atom, Realtek ARM CPUs etc.).

> - Most SBCs (Single Board Computers): Most SBCs (Including Raspberry Pis and **especially the Pi 5**) are too slow to provide a good Jellyfin experience since they often lack proper support for hardware acceleration. If You really want to run Jellyfin on an SBC, please look at models based on the following platforms: Rockchip RK3588 / RK3588S, Intel Core, Intel 12th gen N series

Reviewer: Should probably not use the same phrase twice in one line

Author: How should I change this?

Reviewer 3: Change ```Most SBCs (Single Board Computers):``` to ```Single Board Computers (SBC):```.  It avoids what Ghost mentioned, using the same phrase twice in close proximity, and is more clear since it uses the full phrase before using the abbreviation.

- Intel "Atom" CPUs: Intel J/M/N/Y series low power CPUs up to 11th gen use a different architecture than higher end parts, leading to subpar performance despite what their names might suggest. Please be careful about these parts when shopping for a system to run Jellyfin.
- Prebuilt NAS devices: The software environment on most Prebuilt NAS devices often causes 3rd party software to be hard to install and not work properly. They also often have low end processors that are too slow for a good Jellyfin experience (Intel Atom, Realtek ARM CPUs etc.).

> - Most SBCs (Single Board Computers): Most SBCs (Including Raspberry Pis and **especially the Pi 5**) are too slow to provide a good Jellyfin experience since they often lack proper support for hardware acceleration. If You really want to run Jellyfin on a SBC, please look at models based on the following platforms: Rockchip RK3588 / RK3588S, Intel Core, Intel 12th gen N series

Reviewer 4: ```suggestion
- Most SBCs (Single Board Computers): Most SBCs (Including Raspberry Pis and **especially the Pi 5**) are too slow to provide a good Jellyfin experience since they often lack proper support for hardware acceleration. If You really want to run Jellyfin on an SBC, please look at models based on the following platforms: Rockchip RK3588 / RK3588S, Intel Core, Intel 12th gen N series
```

When using Intel ARC Graphics, Resizable BAR is recommended. Disabling it will result in a 10% reduction in transcoding performance. However, given how fast the media engine on Intel ARC is, this will not be a problem for the vast majority of users.

#### Software HDR to SDR Tone-mapping

> If there is no GPU available, the CPU can also be used to tone-map HDR content to SDR. **This is very demanding on hardware, therefore a GPU is ALWAYS recommended.** When using software tone-mapping, a software encoder will always be used. Tone-mapping 4K 60fps Dolby Vision content to SDR H264 requires a Ryzen 9 5950X for faster than real time transcoding. If you would like to output in H265 or AV1, you may end up in situations where no current desktop CPUs have enough performance.

Reviewer 2: ```suggestion
If there is no GPU available, the CPU can also be used to tone-map HDR content to SDR. **This can be very demanding on CPU, therefore a GPU is ALWAYS recommended.** When using software tone-mapping, a software encoder will always be used. Tone-mapping 4K 60fps Dolby Vision content to SDR H264 requires a Ryzen 9 5950X for faster than real time transcoding. If you would like to output in H265 or AV1, you may end up in situations where no current desktop CPUs have enough performance.
```

Reviewer 2: Ideally this should be moved to a dedicated document about tone mapping but this is good enough for now.
